app/forms/field_handlers/mobile_number_handler.py

```diff
@@ -0,0 +1,24 @@
+from functools import cached_property
+
+from wtforms import StringField
+
+from app.forms.field_handlers.field_handler import FieldHandler
+from app.forms.validators import MobileNumberCheck
+
+
+class MobileNumberHandler(FieldHandler):
+    MANDATORY_MESSAGE_KEY = "MANDATORY_MOBILE_NUMBER"
+
+    @cached_property
+    def validators(self):
+        validate_with = super().validators
+
+        if not self.disable_validation:
```

Comment:
Do we actually use this for anything that doesn't have `options` or textarea?

Reply:
It's available on all the FieldHandlers. I've followed what's available in StringHandler as its basically the same as this bar the validation.